docs/src/docs/asciidoc/user/getting-started.adoc

```diff
@@ -80,23 +112,6 @@ include::{sourcedir33}/clustered/client/src/test/java/org/ehcache/clustered/clie
 
 NOTE: See <<clustered-cache.adoc#,the clustered cache documentation>> for more information on this feature.
 
-==== User managed cache
```

Comment:
Can we have a section id down in the document that matches that old one and links to the user managed doc page? That would make sure we do not break any existing link to it